<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_msg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>FreeSans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:black;
color:white</string>
        </property>
        <property name="text">
         <string>Mensagens ser√£o mostradas aqui!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_video">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>30</y>
           <width>251</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Status Mecanismo</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineedit_mecanismo">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>601</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="label_data">
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>61</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>18</x>
           <y>140</y>
           <width>101</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>ABREPORTA</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>121</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>FECHAPORTA</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>190</y>
           <width>99</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>EXPOEMESA</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>190</y>
           <width>121</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>GUARDAMESA</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>101</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>PARA</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>240</y>
           <width>121</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>CONTROLE</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>300</y>
           <width>101</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>MARCA VIDEO</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineedit_marcavideo">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>360</y>
           <width>771</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>300</y>
           <width>121</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>RESET_MEGA</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_9">
         <property name="geometry">
          <rect>
           <x>322</x>
           <y>140</y>
           <width>111</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ASPECT_RATIO</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineedit_aspect_2">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>140</y>
           <width>91</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>190</y>
           <width>111</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ESCALA</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineedit_escala_2">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>190</y>
           <width>91</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>240</y>
           <width>111</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>PLAY/PAUSE</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_video2">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>399</y>
           <width>151</width>
           <height>101</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_12">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>430</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
